### PR TITLE
Fix WS disconnect race

### PR DESCRIPTION
**Pull request by adamcik**

### CODE REVIEW

**Comment by adamcik** [comment]
Darn, seems I need to do some magic for cross version testing of this with respect to tornado.

**Comment by adamcik** [comment]
Ok, all good now.

File: tests/http/test_handlers.py [hunk 1]
```diff
@@ -35,3 +38,49 @@ def test_static_default_filename(self):
...
+            conn = yield self.connection()
+            conn.stream.close()
+            handlers.WebSocketHandler.broadcast('message')
+
+        @tornado.testing.gen_test
+        def test_broadcast_to_client_without_ws_connection_present(self):
+            yield self.connection()
+            # Tornado checks for ws_connection and raises WebSocketClosedError
+            # if it is missing, this test case simulates winning a race were
+            # this has happened but we have not yet been remove from clients.
```
Comment (jodal) [inline tests/http/test_handlers.py]: s/remove/removed/

**Comment by jodal** [comment]
Looks good to me.

What did this bug look like? A crashed handler in Mopidy and other handlers and new handlers continued working?

**Comment by adamcik** [comment]
Not quite sure, I killed it before I realized what had happened :(

**Comment by adamcik** [comment]
Typo fixed with a rebase this time round.

**Pull request merged**